app/controllers/events/view/sessions/list.js

```diff
@@ -1,6 +1,19 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
 
 export default Controller.extend({
+  ratedSessions: computed('model.data.@each.feedbacks', async function() {
+    let ratedSessions = [];
+    await this.model.data.forEach(async session => {
+      await session.feedbacks.forEach(async feedback => {
+        let { user } = await feedback;
```

Comment:
Write a separate computed property which would return feedbacks and pass it in as a watchable property. In that way, you wouldn't have to use async//await again while looping through feedbacks